.github/workflows/sync.yml

```
@@ -0,0 +1,51 @@
+name: Daily list sync
+
+on:
+  schedule:
+    - cron: "30 15 * * *"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Elixir
+        uses: erlef/setup-elixir@v1
+        with:
+          otp-version: "22.2"
+          elixir-version: "1.9.4"
+
+      - name: Download new list
+        run: curl https://raw.githubusercontent.com/wesbos/burner-email-providers/master/emails.txt -o priv/burner-email-providers/emails.txt
```

Let's use the same script to make sure there's only one way of updating this

```suggestion
        run: ./update_providers.sh
```

---

Fixed.